Branch-SDK-TestBed/src/main/java/io/branch/branchandroidtestbed/SettingsActivity.java

```diff
@@ -35,21 +35,41 @@ void setupRetryEditText() {
 
         retryEditText.setText(Integer.toString(currentRetries));
 
-        retryEditText.setOnEditorActionListener(new TextView.OnEditorActionListener() {
-            @Override
-            public boolean onEditorAction(TextView textView, int i, KeyEvent keyEvent) {
-                if (i == EditorInfo.IME_ACTION_DONE) {
-                    int retries = Integer.valueOf(textView.getText().toString());
-                    PrefHelper.getInstance(SettingsActivity.this).setRetryCount(retries);
-
-                    InputMethodManager imm = (InputMethodManager)getSystemService(SettingsActivity.this.INPUT_METHOD_SERVICE);
-                    imm.hideSoftInputFromWindow(retryEditText.getWindowToken(), 0);
-
-                    Toast.makeText(getApplicationContext(), "Set Network Retries to " + retries, Toast.LENGTH_SHORT).show();
-                    return true;
-                }
-                return false;
+        retryEditText.setOnEditorActionListener((textView, i, keyEvent) -> {
+            if (i == EditorInfo.IME_ACTION_DONE) {
+                int retries = Integer.parseInt(textView.getText().toString());
+                PrefHelper.getInstance(SettingsActivity.this).setRetryCount(retries);
+
+                InputMethodManager imm = (InputMethodManager)getSystemService(INPUT_METHOD_SERVICE);
+                imm.hideSoftInputFromWindow(retryEditText.getWindowToken(), 0);
+
+                Toast.makeText(getApplicationContext(), "Set Network Retries to " + retries, Toast.LENGTH_SHORT).show();
+                return true;
+            }
+            return false;
+        });
+    }
+
+    void setupApiUrlText() {
+        final EditText apiUrlText = findViewById(R.id.api_url_text);
+        final PrefHelper prefHelper = PrefHelper.getInstance(this);
+        String currentApiUrl = prefHelper.getAPIBaseUrl();
+
+        apiUrlText.setText(currentApiUrl);
+
+        apiUrlText.setOnEditorActionListener((textView, i, keyEvent) -> {
+            if (i == EditorInfo.IME_ACTION_DONE) {
+                String newApiUrl = textView.getText().toString();
+                Branch.setAPIUrl(newApiUrl);
+                PrefHelper.setAPIUrl(newApiUrl);
```

Comment:
I see this error when trying to compile

```
SettingsActivity.java:64: error: setAPIUrl(String) is not public in PrefHelper; cannot be accessed from outside package
                PrefHelper.setAPIUrl(newApiUrl);
                          ^
```

Reply:
Ah right. Looks like I accidentally left that line in somehow. Just removed it now, so it should be fixed.